### PR TITLE
Fix for https://bugs.launchpad.net/or/+bug/1959336.

### PR DESCRIPTION
**Pull request by mbm-OR**
Train DPU info, the Load values are unstable.
More info here: https://bugs.launchpad.net/or/+bug/1959336

### CODE REVIEW

**Review by Csantucci — APPROVED** [review]

File: Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs [hunk 1]
```diff
@@ -990,6 +990,12 @@ public string GetDPDebugStatus()
                     {
                         data = (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA;
                     }
+                    if (ThrottlePercent == 0 && data > 0)
+                    {
+                        data = DynamicBrakePercent == -1? (data / MaxForceN) * MaxCurrentA
+                            : DynamicBrakePercent == 0? (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA
+                            : data;
+                    }
```
Comment (Sharpe49) [inline Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs:998]:
```suggestion
                    if (ThrottlePercent >= 0 && DynamicBrakePercent == -1)
                    {
                        data = (data / MaxForceN) * MaxCurrentA;
                    }
                    if (ThrottlePercent == 0 && DynamicBrakePercent >= 0)
                    {
                        data = (data / MaxDynamicBrakeForceN) * DynamicBrakeMaxCurrentA;
                    }
```

Comment (Sharpe49) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs:998]: The above suggestion should replace all of the if conditions (you can't make comments on non edited lines currently in Github).

Comment (Sharpe49) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs:998]: I wanted to make this suggestion in order to simplify the code and avoid a case where you get data = data.

Comment (mbm-OR) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs:998]: You are referring to all the **if conditions** inside the `case CABViewControlUnits.AMPS:` should be  replaced by the suggested code.

Comment (Sharpe49) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs:998]: Yes

Comment (mbm-OR) [reply Source/Orts.Simulation/Simulation/RollingStocks/MSTSDieselLocomotive.cs]: Fix improved, with your code. 
Thank you.

**Review by pzgulyas — APPROVED** [review]
Looks okay

**Pull request merged**